WeakAurasTemplates/WeakAurasTemplates_Wrath.toc

```diff
@@ -0,0 +1,20 @@
+## Interface: 38000
```

Comment:
are we sure that is the right toc?

Reply:
[03:25:33] Dump: value=GetBuildInfo() 
[03:25:33] [1]="3.80.0", 
[03:25:33] [2]="64393", 
[03:25:33] [3]="Nov 11 2025", 
[03:25:33] [4]=38000, 
[03:25:33] [5]="", 
[03:25:33] [6]=" ", 
[03:25:33] [7]=38000